exercises/4.5/manipulacaoDom.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Exercício 4.4</title>
+
+    <style>
+      div {
+        border-color: black;
+        border-style: solid;
+      }
+      .title {
+        text-align: center;
+      }
+
+      .main-content {
+        background-color: yellow;
+      }
+
+      .main-content .center-content {
+        background-color: red;
+        width: 50%;
+        margin: 0 auto;
+      }
+
+      .main-content .center-content p {
+        font-style: italic;
+      }
+    </style>
+  </head>
+  <body>
+    <h1 class="title">Exercício 4.4 - JavaEscripito </h1>
+    <div class="main-content">
+      <div class="center-content">
+        <p>Texto padrão do nosso site</p>
+        <p>-----</p>
+        <p>Trybe</p>
+      </div>
+    </div>
+    <script>
+        /*
+        Aqui você vai modificar os elementos já existentes utilizando apenas as funções:
+        - document.getElementById()
+        - document.getElementsByClassName()
+        - document.getElementsByTagName()
+      (X)  1. Crie uma função que mude o texto na tag <p> para uma descrição de como você se vê daqui a 2 anos. (Não gaste tempo pensando no texto e sim realizando o exercício)
+      (x)  2. Crie uma função que mude a cor do quadrado amarelo para o verde da Trybe.
+      (x) 3. Crie uma função que mude a cor do quadrado vermelho para branco.
+      (x)  4. Crie uma função que corrija o texto da tag <h1>.
+      (x)  5. Crie uma função que modifique todo o texto da tag <p> para maiúsculo.
+        6. Crie uma função que exiba o conteúdo de todas as tags <p> no console.
+        */
+
+        let changeTextPTag = function(text) {
+          let element = document.querySelectorAll("p")[1].innerHTML = text;
+        };
+        changeTextPTag("Casado com filhos e cuidando dos meus pais");
+
+        let changeMainColor = function(color) {
+          let element =  document.querySelector(".main-content").style.background = color;
+        }
+        changeMainColor("#3ba36b");
+
+        let changeCenterMain = function(color) {
+          let element =  document.querySelector(".center-content").style.background = color;
+        }
+        changeCenterMain("#ffffff");
+
+        let modifyH1Text = function(text) {
+          let element =  document.querySelector(".title").innerHTML = text;
+        }
+        modifyH1Text("Exercício 4.5 - Javascript");
+
+        let textToUpperCase = function() {
+          let element = document.getElementsByTagName("p");
+          
+          for( i of element) i.innerHTML = i.innerHTML.toUpperCase();
```

Comment:
Oi Anderson,
não estão faltando as chaves desse for in? Achei pouco prático ler essa linha nessa estrutura

Reply:
Fala Lucas! Muito obrigado, eu nem tinha percebido, pois funcionou da mesma forma! Ja fiz a alteração, vou subir de novo pro GitHub! Vlw